test/skip-subtypes/generated/BodyComplex/README.md

```diff
@@ -1,88 +1,83 @@
 ## An isomorphic javascript sdk for - AutoRestComplexTestService
 
-This package contains an isomorphic SDK for AutoRestComplexTestService.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for AutoRestComplexTestService.
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
+- Node.js version 8.x.x or higher
 - Browser JavaScript
 
 ### How to Install
 
+To use this SDK in your project, you will need to install two packages.
+- `` that contains the client.
+- `@azure/identity` that contains different credentials for you to authenticate the client using Azure Active Directory.
+Install both packages using the below commands.
+Alternatively, you can add these to the dependencies section in your package.json and then run `npm install`.
 ```bash
 npm install
+npm install @azure/identity
 ```
+Please note that while the credentials from the older `@azure/ms-rest-nodeauth` and `@azure/ms-rest-browserauth` packages are still supported, these packages are in maintenance mode receiving critical bug fixes, but no new features.
+We strongly encourage you to use the credentials from `@azure/identity` where the latest versions of Azure Active Directory and MSAL APIs are used and more authentication options are provided.
 
-### How to use
+### How to Use

```

Comment:
This can stay lower case
```suggestion
### How to use

```

Reply:
I addressed this by only capitalizing the first word in all subtitles in the file.